src/test/regress/multi_schedule_hyperscale_superuser

```diff
@@ -19,7 +19,6 @@
 # remove / add node operations, we do not want any preexisting objects before
 # propagate_extension_commands
 # ---
-test: turn_mx_off
```

Comment:
This might break some of the tests when we run hyperscale tests. But, I agree that we should do this. @halilozanakgul do you have any specific concerns on this?

Reply:
The problem is that we currently don't have turn_mx_off.sql file at all. It is not deleted in this PR: it is not currently on main branch!

Reply:
My suggestion is to fix the hyperscale schedules separately from this PR.

Reply:
Yeah, we should have removed this. 😄 I agree with @naisila.
If we have problems while running the tests, we can fix them then.